tests/rafter/before-commit.sh

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
```

Comment:
Already in kyma repo we have a generic makefile with implementation of `before-commit`, so this file should be removed.